### ISSUE TITLE
Unit test blends

### ISSUE THREAD

**Issue by Hoikas** [H-uru/korman#57]
Korman has gotten to the pointer where it is a large, complex programs with many facets that are difficult to naively test. As a first step, we need to begin assembling a library of in-repo packed blend files for the purposes of testing Korman's features.

**Comment by Deledrius** [comment]
Is there a way to automate load & export via blender command line?

**Comment by Hoikas** [comment]
Yes. https://docs.blender.org/api/blender_python_api_master/info_tips_and_tricks.html#don-t-use-blender

**Comment by Deledrius** [comment]
Excellent!  I was unable to find that when I checked before.

**Comment by Hoikas** [comment]
The Blender Python documentation is mostly useless, but there are some hidden nuggets :wink:

**Comment by dpogue** [comment]
Is the plan to have a human-verified set of export PRPs and just run korman on the source files and diff its output PRPs against the verified ones?

**Comment by Hoikas** [comment]
I'm not sure how well comparing PRP diffs would work just because slight differences in the data produce the same result. I'm thinking about kBlendDetail vs kBlendAlpha or objects just being exported in a different order due to code diff noise.

**Comment by dpogue** [comment]
yeah... I vaguely recall diafero had a script for diffing files that had been round-tripped through libHSPlasma
